flocker/provision/_install.py

```diff
@@ -40,9 +40,118 @@
                 "centos/clusterhq-release$(rpm -E %dist).noarch.rpm".format(
                     archive_bucket=ARCHIVE_BUCKET,
                     ),
+    # FLOC-1828 TODO - use ARCHIVE_BUCKET rather than clusterhq-archive-testing
+    # This needs a release to be done to add .deb files to clusterhq-archive
+    'ubuntu-14.04': 'https://clusterhq-archive-testing.s3.amazonaws.com/'
+                'ubuntu/14.04/$(ARCH)'.format(archive_bucket=ARCHIVE_BUCKET),
```

Comment:
The non-effective, but not an error, format is in preparation for the change commented above.  As it is clearly marked with an issue, I think this is OK for now. :question: